### PR TITLE
24704 - Update Type Change Input Section Styling

### PR DESCRIPTION
**Pull request by ArwenQin**
*Issue #:* /bcgov/entity#24704

*Description of changes:*
- When the company name error displays, change hint text of input field to "Change company name"
- When the director error displays, change hint text of input field to "Update directors"
- When the company name or director error displays, change styling of input field to red
- When the company name or director error displays, add a red border to the section (invalid style)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/common/YourCompany/BusinessType.vue [hunk 1]
```diff
@@ -2,11 +2,12 @@
   <div
     id="business-type"
     class="section-container"
-    :class="{'invalid-section': invalidSection}"
+    :class="{'invalid-section': invalidSection || isShowingError}"
   >
```
Comment (ArwenQin) [inline src/components/common/YourCompany/BusinessType.vue:6]: use the invalid-section style for this section before user click "Review and Certify"

**Comment by ArwenQin** [comment]
After updated:
image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-edit-dev--pr-602-jisi7zd3.web.app

**Comment by ArwenQin** [comment]
try: https://business-edit-dev--pr-602-jisi7zd3.web.app/BC0883739/alteration

File: src/components/common/YourCompany/ChangeBusinessType.vue [hunk 2]
```diff
@@ -452,6 +455,20 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
...
 
+  /** Get the input field hint. */
+  get inputHint (): string {
+    if (this.hasAttemptedSubmission) {
+      if (this.nameRequestRequiredError && this.minimumThreeDirectorError) {
+        return 'Change company name and update directors'
+      } else if (this.nameRequestRequiredError) {
+        return 'Change company name'
+      } else if (this.minimumThreeDirectorError) {
+        return 'Update directots'
```
Comment (severinbeauvais) [inline src/components/common/YourCompany/ChangeBusinessType.vue:466]: Typo

Comment (severinbeauvais) [reply src/components/common/YourCompany/ChangeBusinessType.vue:466]: Unless these are _really young_ directors? 😆

**Review by severinbeauvais — APPROVED** [review]
Thanks for the screenshots and test business!

File: src/components/common/YourCompany/ChangeBusinessType.vue (hunk 2 above)
Comment (ArwenQin) [reply src/components/common/YourCompany/ChangeBusinessType.vue]: woops, updated! thx! :)

**Pull request merged**